cards/card_component.js

```diff
@@ -26,6 +31,34 @@ BaseCard["{{componentName}}"] = class extends ANSWERS.Component {
         })
       );
     }
+
+    let feedbackFormSelector = '.js-HitchhikerCard-feedbackForm';
+    let feedbackFormEl = this._container.querySelector(feedbackFormSelector);
+    if (feedbackFormEl) {
+      // For WCAG compliance, the feedback should be a submittable form
+      feedbackFormEl.addEventListener('submit', (e) => {
+        let formTargetEl = e.target;
+        let checkedValue = formTargetEl.querySelector('input:checked').value === 'true';
```

Comment:
nit: would `isChecked` be a little more accurate of a name?

Reply:
It seems like checkedValue indicates which button (thumbs up or down) is clicked, not whether there is a click. So checkedValue is false when a thumbs down is selected

Reply:
ah, got it. Maybe we should rename to `isGood` to be consistent then?

Reply:
Makes sense